cli/src/main/package/ide

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# Check if arguments are given
+if [ -n "${1}" ]; then
+  # Call native ideasy.exe with user-provided arguments
+  ./ideasy "$@"
+fi
+
+UNAME=$(uname)
+
+if [ "$UNAME" == "Linux" ] ; then
+	./ideasy env
+elif [ "$UNAME" == "Darwin" ] ; then
+	./ideasy env
+elif [[ "$UNAME" == CYGWIN* || "$UNAME" == MINGW* ]] ; then
+	./ideasy env --bash
+fi
```

Comment:
```suggestion
if [ "${OSTYPE}" = "cygwin" ] || [ "${OSTYPE}" = "msys" ]; then
  ./ideasy env --bash
else
  ./ideasy env
fi
```
See https://github.com/devonfw/ide/blob/cdf4af5de21489f3fdcbfdab41af858822e0d598/scripts/src/main/resources/scripts/functions#L1785

Please also avoid tabs and use editor compliant with our code-style (e.g. Intellij or configure your editor like Notepad++ accordingly).